src/chunk.js

```diff
@@ -0,0 +1,18 @@
+'use strict'
+
+const isNumber = require('./isNumber')
+
+const chunk = (anArray, chunkSize) => {
+  if (!Array.isArray(anArray) || !isInteger(chunkSize)) return []
```

Comment:
would be simpler by utilizing native Number.isInteger
```suggestion
  if (!Array.isArray(anArray) || !Number.isInteger(chunkSize)) return []
```